Notify/Notify/Notify/ViewModels/NotificationsPageViewModel.cs

```diff
@@ -164,6 +166,13 @@ private NotificationsPageViewModel()
             applyFilterAndSearch();
         }
 
+        private void onMapClicked(Notification notification)
+        {
+            r_Logger.LogInformation($"onMapClicked with: {notification.Name}");
```

Comment:
Better use the notification ID.
The notification's name is not unique (you can use both ID and name, but not only the name).